README.adoc

```diff
@@ -91,51 +93,40 @@ Before you begin, Docker needs to be installed. For installation instructions, r
 Make sure to start your Docker daemon before you proceed.
 
 // =================================================================================================
-// Getting Started
+// Getting started
 // =================================================================================================
 [role='command']
 include::{common-includes}/gitclone.adoc[]
 
 // =================================================================================================
-// Building your application
+// Creating the Dockerfile
 // =================================================================================================
 
-== Building your application
-
-Before you begin, build your application. To do this, navigate to the `start` directory and run the Maven
-`install` goal:
-
-[role='command']
-```
-cd start
-mvn install
-```
-
-Your [hotspot]`pom.xml` file is already configured to add your REST application to the `defaultServer`. But you can tweak this configuration or add your own for another server by updating the [hotspot=execution]`<execution/>` element.
-
-The [hotspot=install-apps]`install-apps` goal copies the application into the specified directory of the specified server.
-In this case, the goal copies the `rest.war` file into the [hotspot=apps]`apps` directory of the `defaultServer` server.
-
-Learn more about this goal on the official https://github.com/WASdev/ci.maven/blob/2.x/docs/install-apps.md[Maven Liberty plug-in repository^].
+== Creating the Dockerfile
 
-pom.xml
-[source, xml, linenums, role="code_column"]
+// File 0
+Dockerfile
+[source, text, linenums, role="code_column"]
 ----
-include::finish/pom.xml[]
+include::finish/Dockerfile[]
 ----
 
-// =================================================================================================
-// Creating the Dockerfile
-// =================================================================================================
-
-== Creating the Dockerfile
+// File 1
+`.dockerignore`
+[source, text, linenums, role="code_column"]
+----
+include::finish/.dockerignore[]
+----
 
+The first step to running your application inside of a Docker container is creating a Dockerfile.
 A Dockerfile is a collection of instructions for building a Docker image that can then be run as a
 container. Every Dockerfile begins with a parent or base image on top of which various commands
-are run. For example, you can start your image from scratch and execute commands that download and
+are run. For example, you can start your image from scratch and run commands that download and
 install Java, or you can start from an image that already contains a Java installation.
 
-[role="code_command hotspot",subs="quotes"]
+Navigate to the `start` directory to begin.
+
+[role="code_command hotspot file=0",subs="quotes"]
 ----
 #Create the `Dockerfile`.#
 `Dockerfile`
```

Comment:
The old guide had the `mvn install` command so that is what the `cd start` command was added to. In this case, there are no commands being run in this section and I'm not sure we should add a command block with only `cd start` in it .

Reply:
I agree no need to add `cd start`